drivers/storage/scaleio/executor/scaleio_executor.go

```diff
@@ -88,6 +89,10 @@ func getLocalVolumeMap() (map[string]string, error) {
 	volMap := map[string]string{}
 	files, err := ioutil.ReadDir(diskIDPath)
 	if err != nil {
+		if os.IsNotExist(err) {
```

Comment:
Hi @codenrhoden,

Be aware of the [tricky nature](http://stackoverflow.com/questions/12518876/how-to-check-if-a-file-exists-in-go/22467409#22467409) of `os.IsNotExist`. The Gotil package has a function named [`FileExists`](https://github.com/akutz/gotil/blob/master/gotil.go#L155-L161) that already handles this.